runners/maven-plugin/src/main/kotlin/DokkaMojo.kt

```diff
@@ -159,6 +156,9 @@ abstract class AbstractDokkaMojo(private val defaultDokkaPlugins: List<Dependenc
     @Parameter
     var failOnWarning: Boolean = DokkaDefaults.failOnWarning
 
+    @Parameter
```

Comment:
IMHO it would be better to annotate it with `@Parameter(defaultValue = "${DokkaDefaults.suppressObviousFunctions}")` I'm not sure how Kotlin default values play with Maven